app/models/external_url.rb

```diff
@@ -0,0 +1,4 @@
+class ExternalUrl < ApplicationRecord
+  belongs_to :refers_to, :polymorphic => true
```

Comment:
The relationship name `resource` would be more consistent with the code base.

`belongs_to :resource, :polymorphic => true`